projects/flow.2D-random-walk/requirements.txt

```diff
@@ -0,0 +1,4 @@
+matplotlib
+numpy
+signac>=1.0.0
+signac-flow>=0.15.0
```

Comment:
I assume this is supposed to be a temporary work-around that is removed prior to merge?

Reply:
Yes.

Reply:
@bdice since we use conda not pip to install the requirements this actually fails as conda tries to find a package from one of its channels that matches this expression.

Reply:
Ten points from Gryffindor for a failed attempt at cleverness. Oh well. We’ll get 0.15 out soon.